crates/plugins/mediator-coordination/src/didcomm/bridge.rs

```diff
@@ -36,25 +37,37 @@ impl DIDResolver for LocalDIDResolver {
             return Ok(Some(self.diddoc.clone()));
         }
 
-        if !did.starts_with("did:key:") {
+        if did.starts_with("did:peer:") {
+            // Add logic to resolve Peer DIDs here
+            let method = DidPeer::new_with_format(PublicKeyFormat::Jwk); // Assuming a Peer DID resolver
+            match method.expand(did) {
+                Ok(diddoc) => Ok(Some(
+                    serde_json::from_value(json!(Document {
```

Comment:
Why need to serialize the Document here?

Reply:
Let me jump in the conversation, cause I've been involved in these changes. There are actually two DID document types here. One from `did-utils` and the other from the `didcomm` dependency. We serialize and deserialize for easy conversion.